internal/handlers/user_test.go

```diff
@@ -1487,9 +1487,10 @@ func (s *UserTestSuite) TestUser_UpdateReadonlyAccess_InternalError() {
 
 func validUser(t *testing.T, id int) *ent.User {
 	t.Helper()
+	name := fmt.Sprintf("User%d", id)
 	return &ent.User{
 		ID:    id,
-		Name:  fmt.Sprintf("User%d", id),
+		Name:  &name,
```

Comment:
Why creating a separate variable instead of assigning value directly in User struct?